### ISSUE TITLE
Add builtin function prototype `MVCC_GET_BY_HEX` in parser

### ISSUE THREAD

**Issue by imtbkcat** [pingcap/parser#798]
## Feature Request
This is sub issue of https://github.com/pingcap/tidb/issues/15938

**Is your feature request related to a problem? Please describe:**

To implement getting mvcc info through SQL, TiDB need to support 4 builtin function.
`MVCC_GET_BY_HEX`  is one of them.

**Describe the feature you'd like:**

add `MVCC_GET_BY_HEX` into parser.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

- Add function name in to `functions.go`
- Add test case.

**Comment by zz-jason** [comment]
<img width="996" alt="Screen Shot 2020-04-07 at 16 20 21" src="https://user-images.githubusercontent.com/5268763/78646512-b9519280-78eb-11ea-968f-262929e3d395.png">

Can we only add one builtin function like `get_mvcc_info(method string, value string)` to handle all these scenarios?

**Comment by imtbkcat** [comment]
@zz-jason yes, it will be much clean. so we could close other three issue, and keep one issue for UCP contributor.

**Comment by hsqlu** [comment]
@zz-jason @imtbkcat If we using the single method `get_mvcc_info`, it means the parameters passed in have to be in order and have different meanings depends on the first method name?  Is it a little bit complicated?

**Comment by zz-jason** [comment]
Actually, there are some MySQL builtin functions like this. I think the complication can be accepted.

**Comment by zz-jason** [comment]
Actually, there are some MySQL builtin functions like this. I think the complication can be accepted.

**Comment by kennytm** [comment]
you could make `get_mvcc_info()` return a table like [`json_table()`](https://dev.mysql.com/doc/refman/8.0/en/json-table-functions.html#function_json-table), or just return a JSON type and let the user to `get_mvcc_info()->>'$.hex'`.
